config/settings.yml

```
@@ -144,6 +144,10 @@ people:
   default_sort: name
   abos: true
   totp_drift: 15
+  cleanup_cutoff_duration:
+    regarding_current_sign_in_at: 1000 # months
+    regarding_roles: 1000 # months
+    regarding_event_participations: 100 # years
```

```suggestion
    regarding_event_participation_years: 100
```

would it make sense to mention the unit (months, years) in the key?